hw/ip/hmac/data/hmac.hjson

```diff
@@ -227,80 +227,89 @@
         }
         { bits: "7:4",
           name: "digest_size",
-          resval: "0x01",
+          resval: "0x08",
           desc: '''Digest size configuration.
 
                 This is a 4-bit one-hot encoded field to select digest size for either HMAC or SHA-2.
-                Invalid values, i.e., values with multiple bits set and value 4'b0000 are mapped in HMAC to SHA2_NONE (4'b0001).
+                Invalid values, i.e., values with multiple bits set (including value 4'b0000) are mapped to SHA2_None.
                 '''
           enum: [
             { value: "1",
-              name: "SHA2_None",
+              name: "SHA2_256",
               desc: '''
-              4'b0001: Unsupported/invalid values and all-zero values are mapped to SHA2_None.
+              4'b0001: SHA-2 256 digest.
               '''
             },
             { value: "2",
-              name: "SHA2_256",
+              name: "SHA2_384",
               desc: '''
-              4'b0010: SHA-2 256 digest.
+              4'b0010: SHA-2 384 digest.
               '''
             },
             { value: "4",
-              name: "SHA2_384",
+              name: "SHA2_512",
               desc: '''
-              4'b0100: SHA-2 384 digest.
+              4'b0100: SHA-2 512 digest.
               '''
             },
             { value: "8",
-              name: "SHA2_512",
+              name: "SHA2_None",
               desc: '''
-              4'b1000: SHA-2 512 digest.
+              4'b1000: Unsupported/invalid values and all-zero values are mapped to SHA2_None.
+              In this case, HMAC/SHA-2 does not start the operation and an error is flagged to SW.
```

Comment:
An error will be flagged if we are in SHA2_None mode and sha_en=1?

Reply:
An error gets flagged only when HMAC is triggered to start and it's in SHA2_None mode. I'll make this clearer in the description.